### PR TITLE
Add a debug mode to PassRunner, which logs out times

### PR DESCRIPTION
**Pull request by kripken**

### CODE REVIEW

**Comment by kripken** [comment]
@jfbastien, I think you were interested in this chrono stuff.

File: src/pass.cpp [hunk 1]
```diff
@@ -66,7 +70,17 @@ void PassRunner::addDefaultOptimizationPasses() {
...
   for (auto pass : passes) {
     currPass = pass;
+    std::chrono::high_resolution_clock::time_point before;
+    if (debug) {
+      std::cerr << "[PassRunner] running pass: " << pass->name << std::endl;
+      before = std::chrono::high_resolution_clock::now();
+    }
     pass->run(this, module);
+    if (debug) {
+      std::chrono::high_resolution_clock::time_point after = std::chrono::high_resolution_clock::now();
```
Comment (jfbastien) [inline src/pass.cpp]: `auto after = std::chrono::high_resolution_clock::now();`

File: src/pass.cpp [hunk 2]
```diff
@@ -66,7 +70,17 @@ void PassRunner::addDefaultOptimizationPasses() {
...
+      std::chrono::duration<double> diff = std::chrono::duration_cast<std::chrono::duration<double>>(after - before);
```
Comment (jfbastien) [inline src/pass.cpp]: `std::chrono::duration<double> diff = after - before;`

**Comment by jfbastien** [comment]
lgtm otherwise.

**Comment by kripken** [comment]
Thanks! Addressed comments, merging.

**Pull request merged**